ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/UserManagementServiceIT.scala

```diff
@@ -216,9 +218,9 @@ final class UserManagementServiceIT extends LedgerTestSuite {
       res3 <- ledger.userManagement.createUser(CreateUserRequest(Some(user2), Nil))
       res4 <- ledger.userManagement.deleteUser(DeleteUserRequest(userId2))
     } yield {
-      assertEquals(res1, user1)
+      assertEquals(res1, CreateUserResponse(Some(user1)))
       assertUserAlreadyExists(res2)
-      assertEquals(res3, user2)
+      assertEquals(res3, CreateUserResponse(Some(user2)))
```

Comment:
Could these changes require to add some exclusions to compatibility tests?

Reply:
Where can I look to check for this? I might not be familiar with the topic

Reply:
Added `Breaks-protobuf: true` to the commit message as per ci/check-protobuf-stability.sh